src/common/task_system/progress_bar.cpp

```diff
@@ -23,11 +31,8 @@ void ProgressBar::finishPipeline() {
         return;
     }
     numPipelinesFinished++;
-    prevCurPipelineProgress = 0.0;
+    prevCurPipelineProgress = -0.01;
     updateProgress(0.0);
-    if (numPipelines == numPipelinesFinished) {
-        resetProgressBar();
-    }
 }
 
 void ProgressBar::updateProgress(double curPipelineProgress) {
```

Comment:
Not related to this PR but can u comment how we control the print frequency here.